### DISCUSSION TITLE
如果把 PPZ.vscode 做成桌面版……

### DISCUSSION THREAD

**Discussion by daGaiGuanYu** [ppz-pro/ppz.vscode#43] [Ideas]
接近俩月没更新版本了……  
已知的问题没修复、打算的特性没添加……  

时间紧是一方面，但是主要是萌生了做桌面版的想法  
前段时间了解了 [Tauri](https://tauri.app/)，感觉这东西太棒了：  
+ **能作为一个原生应用，直接安装在操作系统之上**（而不是 vscode 之上）
+ **同时安装体积不会急剧膨胀**（对比 Electron 应用）
于是当时就决定要做桌面版 PPZ  

但是……Rust，对我来说还是驾驭不了  
不过又了解到基于 Golang 的 [Wails](https://wails.io/)  
所以现在换了决定（决定，也可以换的嘛……）：用 wails 来重新做 PPZ  

桌面版会秉承原来 PPZ 的特色：
+ 轻量（预计安装包不会超过 10M）
+ 简单
+ 面向开发者

所以，，，，你会想要用将来的新 PPZ 吗？

**Comment by wanqqq** [comment]
亲，信件已收到，谢谢！
